### DISCUSSION TITLE
Extension support for language and file icons

### DISCUSSION THREAD

**Discussion by jackTabsCode** [zed-industries/zed#7993] [Feature Requests]
Will file icons be supported as extensions?

Currently I'm working on an extension to provide support for [Luau](https://luau-lang.org). However, there doesn't appear to be any support in the extension API to provide a file icon.

Is it appropriate to make a PR to add a file icon for an unsupported language?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
